tsconfig.json

```diff
@@ -12,13 +12,17 @@
     "preserveConstEnums": true /* Do not erase const enum declarations in generated code. */,
     "removeComments": true /* Do not emit comments to output. */,
 
+    "baseUrl": "src" /* paths are relative to baseUrl */, 
+    "paths": {
+      "semantic-ui-react": ["semanticUiReactReExport"]
+    },
```

Comment:
Ah! I see, that's nice. So basically just a simplification of this path? Or is this something else?